### PR TITLE
fix default Qeue e and webhook startup sequence

### PR DESCRIPTION
**Pull request by QizhouYang**
Signed-off-by: QizhouYang <13523582458@163.com>

### CODE REVIEW

File: pkg/apis/scheduling/v1beta1/types.go [hunk 1]
```diff
@@ -261,6 +261,7 @@ type QueueStatus struct {
 type QueueSpec struct {
 	Weight     int32           `json:"weight,omitempty" protobuf:"bytes,1,opt,name=weight"`
 	Capability v1.ResourceList `json:"capability,omitempty" protobuf:"bytes,2,opt,name=capability"`
+	State QueueState `json:"state,omitempty" protobuf:"bytes,1,opt,name=state"`
```
Comment (k82cn) [inline pkg/apis/scheduling/v1beta1/types.go:264]: That's required to your environment? We already moved this to `QueueStatus` :)

Comment (QizhouYang) [reply pkg/apis/scheduling/v1beta1/types.go]: cache.go file
```
Spec: vcv1beta1.QueueSpec{
			Reclaimable: &reclaimable,
			State:       "Open",
			Weight:      1,
		},
```
declare default Queue！

Comment (QizhouYang) [reply pkg/apis/scheduling/v1beta1/types.go]: If this field `QueueStatus` is not needed，Delete the default value in webhook？

Comment (QizhouYang) [reply pkg/apis/scheduling/v1beta1/types.go]: If this field `State` is not needed，Delete the default value in webhook？

**Pull request merged**